int8 motor_id

bool  motor_onoff
bool  motor_desired_direction 
float64 motor_desired_speed

